server/src/service/PkiService.js

```diff
@@ -154,6 +158,85 @@ exports.getCurrentCARootCert = async function (envId) {
   };
 };
 
+/**
+ * Returns the Environment Status based on configured information for given envId
+ * envId String ID of environment
+ * returns inline_response_200_1
+ **/
+exports.getEnvStatus = async function (envId) {
+  let envStatus = [
+    {
+      phase: 'BUSINESS_SETUP',
+      steps: [
+        {
+          identifier: 'ID_GENERATION',
+          status: 'NOT_STARTED',
+        }
+      ]
+    },
+    {
+      phase: 'TECNICAL_SETUP',
+      steps: [
+        {
+          identifier: 'ENDPOINTS',
+          status: 'NOT_STARTED',
+        },
+        {
+          identifier: 'CSR_EXCHANGE',
+          status: 'NOT_STARTED'
+        },
+        {
+          identifier: 'CERTIFICATE_AUTHORITY',
+          status: 'NOT_STARTED'
+        },
+        {
+          identifier: 'SERVER_CERTIFICATES_EXCHANGE',
+          status: 'NOT_STARTED'
+        },
+        {
+          identifier: 'JWS_CERTIFICATES',
+          status: 'NOT_STARTED'
+        }
+      ]
+    }
+   ];
+   let dfsp = [];
+   try {
+     dfsp = await DFSPModel.findAllByEnvironment(envId).map(r => dfspRowToObject(r));
```

Comment:
must be for the particular dfsp not for all of them

Reply:
Changed per review. Thank you